DogTales/src/dogtales.hpp

```diff
@@ -1,12 +1,15 @@
+#pragma once
 #include <bave/driver.hpp>
-#include <bave/graphics/shape.hpp>
+#include <src/player.hpp>
 
 class DogTales : public bave::Driver {
-	bave::QuadShape m_shape{};
+	std::optional<Player> m_player{};
```

Comment:
At least as of now, this doesn't need to be an optional, and can be initialized in the constructor's member initializer list.